scripts/backup

```diff
@@ -24,68 +24,70 @@ ynh_abort_if_errors
 #=================================================
 # LOAD SETTINGS
 #=================================================
-ynh_print_info "Loading installation settings..."
+ynh_script_progression --message="Loading installation settings..." --time --weight=1
```

Comment:
Note that the --time here is only meant for calibrating the --weight to a different value than 1 (c.f. the progressbar thing) and should be removed after ... Putting `--time --weight=1` is pointless and leads to weird time duration being append during install by actual users...